lib/puppet/type/ec2_launchconfiguration.rb

```diff
@@ -77,3 +92,9 @@ def insync?(is)
   end
 
 end
+def read_only_warning(value, property, should)
```

Comment:
The module already has the ability for marking properties read-only; why this addition?